.github/workflows/k8s-content.yaml

```diff
@@ -0,0 +1,167 @@
+---
+name: Kubernetes content
+
+on:
+  push:
+    branches: [ 'master' ]
```

Comment:
Uhm.... As far as I understand, this will trigger a build each time a patch is pushed to the `master` branch. Which means, once it merges to master. Proposing PRs means pushing to a PR-specific branch.